test/unit_test/pmf_to_pdf_test.cc

```diff
@@ -32,18 +32,20 @@ struct reduction_test_harness
     for (uint32_t i = 0; i < _predictions.size(); i++)
     { ec.pred.a_s.push_back(ACTION_SCORE::action_score{_predictions[i].first, _predictions[i].second}); }
 
-    cout << "\nec.pred.a_s (PMF): " << endl;
-    for (uint32_t i = 0; i < _predictions.size(); i++)
-    { cout << "(" << ec.pred.a_s[i].action << " : " << ec.pred.a_s[i].score << "), " << endl; } }
+    // cout << "\nec.pred.a_s (PMF): " << endl;
```

Comment:
I am commenting these out as I think unit test should not print in general, but leaving in them in in case they are useful for debuggin